holidays/countries/japan.py

```diff
@@ -37,6 +37,9 @@ class Japan(HolidayBase):
     """
 
     country = "JP"
+    special_holidays = {
```

Comment:
Japan has another special holidays (Regarding Emperor of Heisei and Emperor of Reiwa).
And Malaysia also (Malaysia General Election Holiday).

Reply:
The suggestions have been addressed!

Thanks for reviewing this!